### PR TITLE
Add "Description" and "Subject" fields to TokenIssuers lists (SOFTWARE-4915)

### PR DESCRIPTION
**Pull request by matyasselmeci**
Example input:
```
Credentials:
  TokenIssuers:
    - Description: production
      Subject: 7dee38a3-6ab8-4fe2-9e4c-58039c21d817
      URL: https://atlas-auth.web.cern.ch/
      DefaultUnixUser: usatlas1
    - Description: SAM/ETF
      Subject: 5c5d2a4d-9177-3efa-912f-1b4e5c9fb660
      URL: https://atlas-auth.web.cern.ch/
      DefaultUnixUser: usatlas2
    - Description: analysis
      Subject: 750e9609-485a-4ed4-bf16-d5cc46c71024
      URL: https://atlas-auth.web.cern.ch/
      DefaultUnixUser: usatlas3
```
will result in
```
## ATLAS ##
# production:
SCITOKENS /^https\:\/\/atlas\-auth\.web\.cern\.ch\/,7dee38a3\-6ab8\-4fe2\-9e4c\-58039c21d817$/ usatlas1
# SAM/ETF:
SCITOKENS /^https\:\/\/atlas\-auth\.web\.cern\.ch\/,5c5d2a4d\-9177\-3efa\-912f\-1b4e5c9fb660$/ usatlas2
# analysis:
SCITOKENS /^https\:\/\/atlas\-auth\.web\.cern\.ch\/,750e9609\-485a\-4ed4\-bf16\-d5cc46c71024$/ usatlas3
```

### CODE REVIEW

**Comment by matyasselmeci** [comment]
There are no data changes in this PR; I'll make a separate PR for ATLAS mappings once this is deployed.

File: bin/get-scitokens-mapfile [hunk 1]
```diff
@@ -45,19 +45,30 @@ def main(argv):
     for vo_name, vo_data in all_vos_data.vos.items():
         if is_null(vo_data, "Credentials", "TokenIssuers"):
             continue
-        mapfile += f"# {vo_name}\n"
+        mapfile += f"## {vo_name} ##\n"
         for token_issuer in vo_data["Credentials"]["TokenIssuers"]:
             url = token_issuer.get("URL")
+            subject = token_issuer.get("Subject", "")
+            description = token_issuer.get("Description")
```
Comment (brianhlin) [inline bin/get-scitokens-mapfile:52]:
```suggestion
            description = token_issuer.get("Description", "")
```

File: src/schema/vosummary.xsd [hunk 2]
```diff
@@ -187,6 +187,8 @@
                               <xsd:sequence>
                                 <xsd:element name="URL" type="xsd:string"/>
                                 <xsd:element name="DefaultUnixUser" type="xsd:string"/>
+                                <xsd:element name="Description" type="xsd:string"/>
+                                <xsd:element name="Subject" type="xsd:string"/>
```
Comment (brianhlin) [inline src/schema/vosummary.xsd:191]:
```suggestion
                                <xsd:element minOccurs="0" name="Description" type="xsd:string"/>
                                <xsd:element minOccurs="0" name="Subject" type="xsd:string"/>
```

**Review by brianhlin — APPROVED** [review]
Mostly LGTM but I think you've missed a few spots to ensure that the subject/description are optional. Pre-approving

File: src/schema/vosummary.xsd (hunk 2 above)
Comment (matyasselmeci) [reply src/schema/vosummary.xsd:191]: This one's not necessary -- if it's not specified in the yaml, the xml fields still show up, they're just empty.

Comment (brianhlin) [reply src/schema/vosummary.xsd:191]: Oh gotcha

**Pull request merged**